logstash-core/lib/logstash/inputs/metrics.rb

```diff
@@ -39,7 +44,10 @@ def update(time, snapshot)
       #   - We can use a synchronization mechanism between the called thread (update method)
       #   and the plugin thread (run method)
       #   - How we handle back pressure here?
-      @queue << snapshot.to_event
+      #   - one snashot should be only one event
```

Comment:
what I understand from like 48 on is that from one snapshot, so a container of metric_store you generate multiple events, so the line in 47 is misleading. 

Another thing is that the logic to convert to an event should be in the input, not in the snapshot, at my understanding the snapshot is just a container, so should not be aware of format conversions, what do you think?